content/docs/start/model-management/model-cicd.md

```diff
@@ -44,7 +44,7 @@ First, configure the
 (this only needs to be done once):
 
 ```cli
-$ dvc config --global studio.token <your Studio token>
```

Comment:
@yathomasi On this page, I intentionally left it unchanged since it is about CI, where you don't want the interactive experience of `dvc studio login`.